plugins/GSdx/GSDeviceOGL.cpp

```diff
@@ -1242,12 +1242,12 @@ void GSDeviceOGL::CopyRect(GSTexture* sTex, GSTexture* dTex, const GSVector4i& r
 #ifdef ENABLE_OGL_DEBUG
 	PSSetShaderResource(6, sTex);
 #endif
-
-	glCopyImageSubData( sid, GL_TEXTURE_2D,
-			0, r.x, r.y, 0,
-			did, GL_TEXTURE_2D,
-			0, 0, 0, 0,
-			r.width(), r.height(), 1);
+	if (GLLoader::found_GL_ARB_copy_image && glCopyImageSubData)
```

Comment:
yea because like it is now it doesn't have , just making sure If I should add or not when changing.